tools/keepalive-tool.c

```diff
@@ -155,6 +156,10 @@ int main(int argc, char **argv)
     if (!g_option_context_parse(ctx, &argc, &argv, &error)) {
         failure("Option parsing: %s", error->message);
     }
+    if (argc > 0 && strcmp(argv[1], "--") == 0) {
```

Comment:
AFAICT needs to be: if (argc > 1) ...

argv[argc] == NULL -> argc == 1 -> strcmp(argv[1], "--") == strcmp(NULL, "--")